src/loaders/Loader.js

```diff
@@ -1,398 +1,398 @@
-
-			if ( m.vertexColors !== undefined ) {
-
-				if ( m.vertexColors === 'face' ) {
-
-					mpars.vertexColors = THREE.FaceColors;
-
-				} else if ( m.vertexColors ) {
-
-					mpars.vertexColors = THREE.VertexColors;
-
-				}
-
-			}
-
-			// colors
-
-			if ( m.colorDiffuse ) {
-
-				mpars.color = rgb2hex( m.colorDiffuse );
-
-			} else if ( m.DbgColor ) {
-
-				mpars.color = m.DbgColor;
-
-			}
-
-			if ( m.colorSpecular ) {
-
-				mpars.specular = rgb2hex( m.colorSpecular );
-
-			}
-
-			if ( m.colorEmissive ) {
-
-				mpars.emissive = rgb2hex( m.colorEmissive );
-
-			}
-
-			// modifiers
-
-			if ( m.transparency !== undefined ) {
-
-				console.warn( 'THREE.Loader: transparency has been renamed to opacity' );
-				m.opacity = m.transparency;
-
-			}
-
-			if ( m.opacity !== undefined ) {
-
-				mpars.opacity = m.opacity;
-
-			}
-
-			if ( m.specularCoef ) {
-
-				mpars.shininess = m.specularCoef;
-
-			}
-
-			// textures
-
-			if ( m.mapDiffuse && texturePath ) {
-
-				create_texture( mpars, 'map', m.mapDiffuse, m.mapDiffuseRepeat, m.mapDiffuseOffset, m.mapDiffuseWrap, m.mapDiffuseAnisotropy );
-
-			}
-
-			if ( m.mapLight && texturePath ) {
-
-				create_texture( mpars, 'lightMap', m.mapLight, m.mapLightRepeat, m.mapLightOffset, m.mapLightWrap, m.mapLightAnisotropy );
-
-			}
-
-			if ( m.mapAO && texturePath ) {
-
-				create_texture( mpars, 'aoMap', m.mapAO, m.mapAORepeat, m.mapAOOffset, m.mapAOWrap, m.mapAOAnisotropy );
-
-			}
-
-			if ( m.mapBump && texturePath ) {
-
-				create_texture( mpars, 'bumpMap', m.mapBump, m.mapBumpRepeat, m.mapBumpOffset, m.mapBumpWrap, m.mapBumpAnisotropy );
-
-			}
-
-			if ( m.mapNormal && texturePath ) {
-
-				create_texture( mpars, 'normalMap', m.mapNormal, m.mapNormalRepeat, m.mapNormalOffset, m.mapNormalWrap, m.mapNormalAnisotropy );
-
-			}
-
-			if ( m.mapSpecular && texturePath ) {
-
-				create_texture( mpars, 'specularMap', m.mapSpecular, m.mapSpecularRepeat, m.mapSpecularOffset, m.mapSpecularWrap, m.mapSpecularAnisotropy );
-
-			}
-
-			if ( m.mapAlpha && texturePath ) {
-
-				create_texture( mpars, 'alphaMap', m.mapAlpha, m.mapAlphaRepeat, m.mapAlphaOffset, m.mapAlphaWrap, m.mapAlphaAnisotropy );
-
-			}
-
-			//
-
-			if ( m.mapBumpScale ) {
-
-				mpars.bumpScale = m.mapBumpScale;
-
-			}
-
-			if ( m.mapNormalFactor ) {
-
-				mpars.normalScale = new THREE.Vector2( m.mapNormalFactor, m.mapNormalFactor );
-
-			}
-
-			var material = new THREE[ mtype ]( mpars );
-
-			if ( m.DbgName !== undefined ) material.name = m.DbgName;
-
-			return material;
-
-		};
-
-	} )()
-
-};
-
-THREE.Loader.Handlers = {
-
-	handlers: [],
-
-	add: function ( regex, loader ) {
-
-		this.handlers.push( regex, loader );
-
-	},
-
-	get: function ( file ) {
-
-		for ( var i = 0, l = this.handlers.length; i < l; i += 2 ) {
-
-			var regex = this.handlers[ i ];
-			var loader  = this.handlers[ i + 1 ];
-
-			if ( regex.test( file ) ) {
-
-				return loader;
-
-			}
-
-		}
-
-		return null;
-
-	}
-
-};
+/**
```

Comment:
Mixed line ending in the repository. This file uses CR-LF (Windows) line ending except for a single line that uses LF (Unix) that comes from a different commit.

I wonder if this repository should have a `.gitattributes`: https://help.github.com/articles/dealing-with-line-endings/